setup.py

```diff
@@ -2,12 +2,12 @@
 
 setup(
     name="bulk",
-    version="0.0.1",
+    version="0.0.2",
     packages=find_packages(),
-    install_requires=["typer>=0.4.1", "bokeh>=2.4.3"],
+    install_requires=["typer>=0.4.1", "bokeh>=2.4.3", "matplotlib>=3.4.2"],
```

Comment:
Why is matplotlib a required dependency now?

Reply:
My mistake